src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -19,9 +19,16 @@ message ApplicationInformationResponse {
 
 ////////////////////////////////////////////
 
-message ResourceCommandRequest {
+// Defines a command that may be invoked on a resource.
+// Defined by the resource service and sent to the dashboard.
+// When a command is to be executed, an instance of ResourceCommandRequest is constructed
+// using data from this message.
+message ResourceCommand {
+    // Unique identifier for the command. Not intended for display.
     string command_type = 1;
-    string name = 2;
+    // The display name of the command, to be shown in the UI. May be localized.
+    string display_name = 2;
+    // The unique name of the resource type. Matches ResourceType.unique_name and Resource.resource_type.
     string resource_type = 3;
```

Comment:
What's the purpose of `resource_type` on the `ResourceCommand` message now? I understand it on `ResourceCommandRequest` (to ensure uniqueness of `resource_name` in all scenarios) but since `ResourceCommand` is delivered as part of either `ResourceType.commands` or `Resource.commands`, I'm not sure I understand the intent.

Reply:
I guess it can be inferred. Shall we remove it?

Reply:
If there's no obvious reason for it, then probably better to remove it? I assume its easier to add things later if we realize we do need it than remove it if we realize we don't after implementations expect it.